lib/urn.js

```diff
@@ -50,87 +52,13 @@ module.exports = {
     }
     for (var index in urns) {
       var urn = urns[index];
-      if (!urn.match(allowedProtocolsRegExp) && urn.match(/^([^:]+:)+[^:]+$/)) {
+      var validUrnRegExp = /^([^:]+:)+[^:]+$/;
+      if (urn.match(validUrnRegExp)) {
         return true;
-      }
-
-      // Parse ID portion of URN
-      var cat = urn.split(':').slice(0, -1)
-      var id = urn.split(':').slice(-1)[0];
-
-      if (id.length % 2 !== 0) {
-        return 'Identity must have even length and be in proper format';
-      }
-
-      // Split identity by each byte
-      var charArr = id.match(/.{1,2}/g);
-
-      // Match category of URN
-      switch (cat) {
```

Comment:
instead of deleting it, I've just commented out the stricter validation for now in case we decide to reintroduce it in the future